docs/sql-migration-guide.md

```diff
@@ -30,6 +30,7 @@ license: |
 - Since Spark 3.5, the `plan` field is moved from `AnalysisException` to `EnhancedAnalysisException`.
 - Since Spark 3.5, `spark.sql.optimizer.canChangeCachedPlanOutputPartitioning` is enabled by default. To restore the previous behavior, set `spark.sql.optimizer.canChangeCachedPlanOutputPartitioning` to `false`.
 - Since Spark 3.5, the `array_insert` function is 1-based for negative indexes. It inserts new element at the end of input arrays for the index -1. To restore the previous behavior, set `spark.sql.legacy.negativeIndexInArrayInsert` to `true`.
+- Since Spark 3.5, the default value of `spark.sql.maxSinglePartitionBytes` is changed from `Long.MaxValue` to `128m`.
```

Comment:
It's too late for Apache Spark 3.5.0 because we are on RC2 already. Please target this 4.0, @pan3793 .

Reply:
Please add a sentence, `To restore the previous behavior ...`, additionally to be clear.

Reply:
The sentence clearly says we change the default value of conf `xyz` from `A` to `B`, seems too redundant with `To restore the previous behavior, set xyz=A`

Reply:
Moved to `3.5 to 4.0`